psi4/src/psi4/libfunctional/superfunctional.cc

```diff
@@ -381,6 +384,48 @@ void SuperFunctional::add_c_functional(std::shared_ptr<Functional> fun) {
     can_edit();
     c_functionals_.push_back(fun);
 }
+void SuperFunctional::copy_density_tolerance(double cut) {
```

Comment:
This isn't used as far as I can tell? Should be handled by the `set` function.

Reply:
right, I think it was needed before when I did things incorrectly but can be removed now.